package.json

```diff
@@ -25,7 +25,8 @@
     "travis:lint": "npm run lint && npm run security",
     "travis:test": "npm run test -- --runInBand",
     "appveyor:test": "npm run test",
-    "defaults": "webpack-defaults"
```

Comment:
Not entirely sure what happened with this... something weird must have happened at npm install. I'll fix.